src/BaseElement.php

```diff
@@ -386,6 +378,34 @@ public function isVoidElement(): bool
         ]);
     }
 
+    /**
+     * Dynamically handle calls to the class.
+     * Check for methods finishing by If or fallback to Macroable.
+     *
+     * @param  string  $method
+     * @param  array   $parameters
+     * @return mixed
+     *
+     * @throws \BadMethodCallException
+     */
+    public function __call($name, $arguments)
+    {
+        if (Str::endsWith($name, 'If')) {
```

Comment:
Use a helper function (`ends_with`?) instead of the facade here

Reply:
done